lib/gui/os/dropzone/directives/dropzone.js

```diff
@@ -41,7 +41,7 @@ module.exports = ($timeout) => {
       osDropzone: '&'
     },
     link: (scope, element) => {
-      const domElement = element[0];
+      const domElement = _.first(element);
```

Comment:
Oh, right – in that case renaming it to `$element` could clarify it. What do you think?